independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Methods.java

```diff
@@ -115,7 +120,9 @@ static boolean isObjectToString(MethodInfo method) {
 
     static void addInterceptedMethodCandidates(BeanDeployment beanDeployment, ClassInfo classInfo,
             Map<MethodKey, Set<AnnotationInstance>> candidates,
-            List<AnnotationInstance> classLevelBindings) {
+            List<AnnotationInstance> classLevelBindings, Consumer<BytecodeTransformer> bytecodeTransformerConsumer) {
+
+        Set<String> methodsFromWhichToRemoveFinal = new HashSet<>();
```

Comment:
This could potentially remove `final` from more methods than intended, but that shouldn't be a problem, right?